src/pretix/control/forms/filter.py

```diff
@@ -363,6 +363,28 @@ class OrderSearchFilterForm(OrderFilterForm):
         )
     )
 
+    status = forms.ChoiceField(
```

Comment:
Do we really need to copy-paste this? OrderSearchFilterForm and EventOrderFilterForm both depend on OrderFilterForm -- can we do this there instead and only have it once?